src/css/profile/mobile/common/tizen.less

```diff
@@ -12,13 +12,14 @@
 @unit_base: (360 / 720) * @px_base;
 
 .font(light) {
-	font-family: Roboto-Light, sans-serif-light;
+	font-family: sans-serif-light;
 }
 .font(regular) {
-	font-family: Roboto-Regular, sec-roboto-light;
+	font-family: sec-roboto-light;
+	font-weight: normal;
 }
 .font(medium) {
-	font-family: Roboto-Medium, sec-roboto-light;
+	font-family: sec-roboto-light;
```

Comment:
I though that sec-robot-light is alternative font if the Roboto-Medium is not available